lib/rubygems/commands/update_command.rb

```diff
@@ -146,18 +148,18 @@ def highest_installed_gems # :nodoc:
     hig
   end
 
-  def highest_remote_version(spec) # :nodoc:
+  def highest_remote_tuple(spec) # :nodoc:
```

Comment:
maybe specify in the name a tuple of what?

Reply:
I like it, I'll rename this helper :+1:

Reply:
This returns a `NameTuple` instance, probably not the best name since it also includes version and platform information. I guess I could name it `highest_remote_name_tuple`, since resolving the naming issue is probably out of the scope of this PR?

Reply:
Since you are renaming from `highest_remote_version` to `highest_remote_tuple` i think is of the scope of this PR but i don't have a suggestion for a better name so lets keep it for now 😁

Reply:
Yeah, what I mean is that if I name it `highest_remote_name_version_platform_tuple`, it might be more clear, but it makes it inconsistent with the class it returns (an instance of the `NameTuple` class). So I'm not sure it's worth it.

But maybe naming it `highest_remote_name_tuple` clarifies a bit that the tuple is about how we identify the gem?

Reply:
I can at least add this rename to #2643 if you agree!

Reply:
Ahh i see, thanks for explaining, yeah lets add the name in #2643 👍